packages/larva-patterns/modules/article-callout/article-callout.twig

```diff
@@ -0,0 +1,3 @@
+<p class="article-callout larva // {{ article_callout_classes }}">
+        <span class="article-callout-title {{ article_callout_title_classes }}">{ article_callout_title }</span> | <span class="article-callout-text {{ article_callout_text_classes }}">{ article_callout_text }</span>
```

Comment:
Have you checked this in a screen reader? I'm not sure if the pipe character is read aloud - that can also be added via the CSS `content` property if it's an issue. The algorithm `a-icon-after` can help with that.

Also `{ article_callout_text }`: 
1. should have two brackets to be converted properly to PHP e.g. {{ article_callout_text }}
2. should probably use `_markup` because it's possible this would include an anchor or other HTML tags since in Gutenberg I'm guessing it can be rich text

Reply:
Is there an example of `a-icon-after` being implemented that I can take a look at? 

2. Where can I find `_markup` in the codebase? I have tried searching this repo and the pmc-larva plugin and can't find it.

Reply:
Sure - you could add a variation lrv-a-icon-pipe like this: https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva-css/src/02-algorithms/a-icon/a-icon.common.inline.scss#L74

And an example of the classes in use is the envelope icon next to `Newsletters` https://www.artnews.com/ - they can be added as `lrv-a-icon-(before/after) lrv-a-icon-(name)`

Reply:
An example of using the _markup keyword is in here, in c-dek: https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva-patterns/components/c-dek/c-dek.twig#L7